components/server/src/container-module.ts

```diff
@@ -178,9 +177,8 @@ export const productionContainerModule = new ContainerModule((bind, unbind, isBo
         const config = ctx.container.get<Config>(Config);
         return { address: config.imageBuilderAddr };
     });
-    bind(CachingImageBuilderClientProvider).toSelf().inSingletonScope();
-    bind(WorkspaceClusterImagebuilderClientProvider).toSelf().inSingletonScope(); // during the transition period, we have two kinds of image builder client providers
-    bind(ImageBuilderClientProvider).toService(CachingImageBuilderClientProvider);
+    bind(WorkspaceClusterImagebuilderClientProvider).toSelf().inSingletonScope();
+    bind(ImageBuilderClientProvider).toService(WorkspaceClusterImagebuilderClientProvider);
```

Comment:
this changes the client type of any `@Inject(ImageBuilderClientProvider)` to the workspace cluster client provider, instead of the application cluster client provider (`CachingImageBuilderClientProvider`)